gateway/src/test/java/io/camunda/zeebe/gateway/grpc/GrpcErrorMapperTest.java

```diff
@@ -30,11 +30,8 @@
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.parallel.Execution;
-import org.junit.jupiter.api.parallel.ExecutionMode;
 import org.slf4j.helpers.NOPLogger;
 
-@Execution(ExecutionMode.CONCURRENT)
```

Comment:
Sorry but is this related to the other changes? If not I would suggest that we next time create another PR for it, even if this change is so small.

Reply:
Oh, it is in another PR, I'm not sure why it's there :thinking: In fact we already merged that PR...